IBAnimatable/IBAnimatable tvOS/IBAnimatable tvOS.h

```diff
@@ -0,0 +1,19 @@
+//
+//  IBAnimatable tvOS.h
+//  IBAnimatable tvOS
+//
+//  Created by Steven on 6/9/17.
+//  Copyright © 2017 IBAnimatable. All rights reserved.
+//
+
+#import <UIKit/UIKit.h>
+
+//! Project version number for IBAnimatable tvOS.
+FOUNDATION_EXPORT double IBAnimatable_tvOSVersionNumber;
```

Comment:
No, I think I can remove the whole `IBAnimatable tvOS` & `IBAnimatable tvOSTests` folders